app.py

```diff
@@ -2050,7 +2589,7 @@ def refreshDatabase():
     # password = input("Password: ")
 
     username = 'root'
-    password = 'blahblah'
+    password = 'pavani@17'
 
```

Comment:
Don't change this password, please.

Reply:
My Computer Password is different :(
Anyway we need to take input right?